Frameworks/include/CoreTextInternal.h

```diff
@@ -76,38 +76,20 @@ struct __CTFrame : CoreFoundation::CppBase<__CTFrame> {
 #pragma region CTRun
 struct __CTRun : CoreFoundation::CppBase<__CTRun> {
     __CTRun()
-        : _attributes(CFDictionaryCreateMutable(kCFAllocatorSystemDefault,
-                                                0,
-                                                &kCFCopyStringDictionaryKeyCallBacks,
-                                                &kCFTypeDictionaryValueCallBacks)),
+        : _attributes(woc::MakeStrongCF(CFDictionaryCreateMutable(kCFAllocatorSystemDefault,
```

Comment:
It is slightly more concise and involves fewer temporary objects to simply use `woc::TakeOwnership`